src/main/java/fruitshop/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package fruitshop.service.impl;
+
+import fruitshop.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String file) {
+        List<String> data = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(file))) {
```

Comment:
Make sure to use the try-with-resources statement for the `BufferedReader` to ensure that the resource is closed properly after use, which is correctly implemented here.